Source/JavaScriptCore/yarr/RegularExpression.h

```diff
@@ -29,14 +29,14 @@
 
 namespace JSC { namespace Yarr {
 
-enum MultilineMode { MultilineDisabled, MultilineEnabled };
-enum TextCaseSensitivity { TextCaseSensitive, TextCaseInsensitive };
-enum UnicodeMode { UnicodeUnawareMode, UnicodeAwareMode };
+enum class MultilineMode : bool { Disabled, Enabled };
+enum class TextCaseSensitivity : bool { Sensitive, Insensitive };
```

Comment:
It would be neat if there were a way to avoid repeating (forms of) the word "sensitive"... that said, I suppose `CaseSensitive::{Yes,No}` or `IgnoreCase::{No,Yes}` might make the fact that "sensitive" is the default more confusing. Maybe the word `Text` could be dropped, at least?